common/main.conf

```diff
@@ -1,13 +1,18 @@
 # Log notice level and higher (e.g. state cache
 # emits useful log messages on notice level).
-error_log stderr notice;
+error_log stderr info;
```

Comment:
Discussed in person. The conclusion (correct me if I am wrong) is that it does not change much the amount of data logged and that the issue will be addressed as soon as it's technically possible (we will have to drop MesosDNS and switch to direct API queries which is not trivial and has some dependencies on its own).